src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/choose-your-aggregation-method.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Choose your aggregation method
+tags:
+  - Alerts and Applied Intelligence
+  - New Relic Alerts
+metaDescription: "Use this for guidance on choosing the best aggregation method for your alerting conditions and notification thresholds."
+---
+
+Alerts conditions provide a sophisticated set of tools for describing when you want to be notified about something that's happened or failed to happen on something you're monitoring. For best results, choose your aggregation method that best matches the way your data arrives.
```

Comment:
Should this be `Alert conditions"?

Reply:
I try to avoid this phrasing because technically it should be "Alerts conditions" and then you get into this weird double plural situation.